### PR TITLE
[F/E] Add cookie policy

### PR DESCRIPTION
**Pull request by shirve**
### Changes
Added cookie policy banner

### Preview
Screenshot 2022-11-24 at 16 59 35

### CODE REVIEW

File: team-fingerprints-web/src/components/CookiePolicy/CookiePolicy.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import Cookies from "js-cookie";
+import { ReactComponent as XCircle } from "assets/XCircle.svg";
```
Comment (tobob) [inline team-fingerprints-web/src/components/CookiePolicy/CookiePolicy.tsx:2]: order of imports:
```
main libraries - react, react-dom
3rd party libraries
our libraries
components
svgs/icons
styles
```

File: team-fingerprints-web/src/components/CookiePolicy/CookiePolicy.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+import "./styles.sass";
+import { useState } from "react";
+
+const cookiePolicyUrl = process.env.COOKIE_POLICY_URL;
+
+const cookiePolicy = Cookies.get("cookiePolicy");
+
+const CookiePolicy = () => {
+  const [isVisible, setIsVisible] = useState(true);
```
Comment (tobob) [inline team-fingerprints-web/src/components/CookiePolicy/CookiePolicy.tsx:12]: please change into:

const [isVisible, setIsVisible] = useState(Cookies.get("cookiePolicy"));

to remove unnecessary const `cookiePolicy`

**Review by tobob — CHANGES_REQUESTED** [review]
I have some feedback ;)

**Review by tobob — APPROVED** [review]

**Comment by Matesty** [comment]
@shirve 
- for now without redirection to the cookie policy?

**Comment by shirve** [comment]
@Matesty 
Correct. The url for cookie policy was not yet provided.

**Review by Matesty — APPROVED** [review]

**Comment by tobob** [comment]
@shirve please resolve conflicts my merging with main

**Pull request merged**